src/probe_matcher.cpp

```diff
@@ -557,4 +567,24 @@ void ProbeMatcher::list_structs(const std::string& search)
     std::cout << match << std::endl;
 }
 
+// Transform the kernel syntax (function [module]) into bpftrace syntax
+// (module:function).
+void ProbeMatcher::adjust_kernel_modules(
+    std::unique_ptr<std::istream>& symbol_list) const
```

Comment:
This should return a new istream and not modify a unique_ptr taken by reference - unique_ptr& is a pretty unidiomatic construct in the first place.